sickrage/core/__init__.py

```diff
@@ -415,14 +419,39 @@ def start(self):
             id="PROPERSEARCHER"
         )
 
+        now_second = datetime.datetime.now().second
+        second_start = now_second + 5
+        minute_start = datetime.datetime.now().minute
+        if second_start > 59:
+           second_start = second_start - 60
+           minute_start = minute_start + 1 
+           date_start = datetime.datetime.now().replace(minute=minute_start, second=second_start)
+        else:
+           date_start = datetime.datetime.now().replace(second=second_start)
+
         # add trakt.tv checker job
         self.srScheduler.add_job(
             self.TRAKTSEARCHER.run,
-            srIntervalTrigger(**{'hours': 1}),
+            srIntervalTrigger(**{'hours': 1,
+                                 'start_date': date_start
+                                }
+                             ),
             name="TRAKTSEARCHER",
             id="TRAKTSEARCHER"
         )
 
+        # add trakt.tv rolling download job
+        self.srScheduler.add_job(
+            self.TRAKTROLLING.run,
+            srIntervalTrigger(**{'minutes': self.srConfig.TRAKT_ROLLING_FREQUENCY, 
+                                 'min': self.srConfig.MIN_TRAKTROLLING_FREQ,
+                                 'start_date': date_start
```

Comment:
I need to start immediately this 2 job because we need self.trakt_api evalued.
 Add a line note